crates/test-framework/src/metrics/mod.rs

```diff
@@ -400,24 +407,24 @@ impl<T: ExtendedMetrics, R: ExtendedMetrics> QueryMetrics<T, R> {
         let mut base_fields = vec![
             Field::new("run_id", DataType::Utf8, false),
             Field::new("spiced_version", DataType::Utf8, false),
-            Field::new("started_at", DataType::Int64, false),
-            Field::new("finished_at", DataType::Int64, false),
+            Field::new("started_at", DataType::UInt64, false),
+            Field::new("finished_at", DataType::UInt64, false),
             Field::new("query_name", DataType::Utf8, false),
             Field::new("status", DataType::Utf8, false),
-            Field::new("min_duration_ms", DataType::Int64, false),
-            Field::new("max_duration_ms", DataType::Int64, false),
-            Field::new("iterations", DataType::Int32, false),
+            Field::new("min_duration_ms", DataType::UInt64, false),
+            Field::new("max_duration_ms", DataType::UInt64, false),
+            Field::new("iterations", DataType::UInt64, false),
             Field::new("commit_sha", DataType::Utf8, false),
             Field::new("branch_name", DataType::Utf8, false),
         ];
 
         base_fields.extend(extended_fields);
 
         base_fields.extend(vec![
```

Comment:
We can put the extended fields at the end - I had them split up like this to match the existing metrics table format, but since we don't need to do that anymore we can add them at the end.